### DISCUSSION TITLE
How do I implement 44x44 touch area for touch devices

### DISCUSSION THREAD

**Discussion by ritz078** [adobe/react-spectrum#5593] [Q&A]
According to a11y guidelines, the touch area on touch enabled devices should be minimum 44x44. I tried using the approach mentioned here: https://x.com/adamwathan/status/1735785489519534406?s=20

The problem is that whenever I click on the area outside, the `onPress` even of buttons do not get triggered. 

https://github.com/adobe/react-spectrum/assets/5389035/85a54866-2c5a-4a2f-b24e-596c4642d98f

In the above video, you can see that clicking in the red region doesn't trigger anything.

Any idea how to approach this issue ?

**Comment by snowystinger** [comment]
Probably related to https://github.com/adobe/react-spectrum/issues/2506
